File: ATS.UI/ClientApp/src/app/core/interfaces/interviewSchedule.ts
export class IInterviewScheduleDisplay {
    interviewScheduleID: number;
    reviewQueueID: number;
    reviewStatusID: number;
    reviewer?: string;
    dateInterview: Date;
    duration: number;
    interviewTypeID: number;
    interviewType?: string;
    timeZone: string;
    comment: string;
    status?: string;
    candidate?: string;
    requirement?: string;
    recruiter?: string;
}

export class IInterviewSchedule {
    InterviewScheduleID: number;
    ReviewQueueID: number;
    ReviewStatusID: number;
    DateInterview?: string;
    Duration: number;
    InterviewTypeID: number;
    TimeZone: string;
    Comment: string;
    DateCreated?: Date
    CreatedBy?: string;
    DateModified?: Date;
    ModifiedBy?: string;
    CommentBy?: number;
}

File: ATS.DTO/InterviewSchedule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ATS.DTO
{
    public class InterviewSchedule : AbstractDataObject
    {
        public int InterviewScheduleID { get; set; }
        public int ReviewQueueID { get; set; }
        public int ReviewStatusID { get; set; }
        public DateTime? DateInterview { get; set; }
        public int Duration { get; set; }
        public int InterviewTypeID { get; set; }
        [Ignore]
        public string InterviewType { get; set; }
        public string TimeZone { get; set; }
        public string Comment { get; set; }
        public int CommentBy { get; set; }
        [Ignore]
        public string Reviewer { get; set; }

        [Ignore]
        public string Status { get; set; }
        [Ignore]
        public string Candidate { get; set; }
        [Ignore]
        public string Requirement { get; set; }
        [Ignore]
        public string Recruiter { get; set; }
    }
}
